package Utils.CustomExceptions;

public class AlreadyExistingUser extends Throwable{

}
